Earthfile

```diff
@@ -1,4 +1,4 @@
-VERSION 0.7
+VERSION --try 0.7
```

Comment:
Is --try required?

Reply:
Yes, its a feature flag to enable try - finally block